Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -55,6 +55,14 @@ - (NSString *)appSecret {
 #endif
 }
 
+-(NSString *)appSecretAAD {
+#if GCC_PREPROCESSOR_MACRO_PUPPET
+  return kMSPuppetAADAppSecret;
+#else
+  return kMSSwiftObjcAADAppSecret;
```

Comment:
This will mislead things while testing it. We decided not to implement AAD in Objective C app, this will send all logs to Swift app in App Center, it would be better to display an alert that we don't support AAD in Sasquatch ObjC app.

Reply:
Synced with @manuchandel, we will keep this as it is.